extensions/cornerstone/src/commandsModule.ts

```diff
@@ -604,6 +609,36 @@ function commandsModule({
       );
       toolGroup.setToolEnabled(ReferenceLinesTool.toolName);
     },
+    setDisplaySetToViewport: ({ displaySet }) => {
+      const cachedDisplaySetKeys = [
+        ...displaySetService.getDisplaySetCache().keys(),
+      ];
+      const displaySetKey = Object.keys(displaySet)[0];
+
+      // Check to see if computed display set is already in cache
+      if (!cachedDisplaySetKeys.includes(displaySetKey)) {
+        displaySetService._addDisplaySetsToCache([displaySet[displaySetKey]]);
+      }
```

Comment:
you should use getDIsplaySetByUID

Reply:
I'm getting the display set keys of the display sets in cache in order to check if I need to add the current display set set to cache, I'm not getting any of the display sets to use so I don't think that would work in this situation.

Reply:
Removed _addDisplaySetsToCache and replaced it with setting the display set to displaySetCache